sys-libs/lwp/lwp-2.8.ebuild

```diff
@@ -23,5 +21,5 @@ src_configure() {
 
 src_install() {
 	default
-	prune_libtool_files
+	find "${D}" -name '*.la' -delete || die
```

Comment:
`${D}` is technically fine for this, but `${ED}` is more precise and what the [policy guide example](https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0303) uses, so I'll change this on merge.

Reply:
Thanks, I'll keep an eye on this for the future :)